_posts/platform/internals/2000-01-01-routing.md

```diff
@@ -1,14 +1,18 @@
 ---
 title: Routing
-modified_at: 2018-06-15 00:00:00
+modified_at: 2019-07-24 00:00:00
 tags: internals reverse-proxy routing http https timeout websocket
 index: 3
 ---
 
 ## Requests scheduling
 
-Each of our front web servers is applying a strict round-robin scheduling over the different
-containers of your application.
+Each of our front web servers is applying a strict round-robin scheduling over
+the different containers of your application.
+
+Our frontend servers also include a quarantine mechanism: if one of your
+containers unexpectedly cut a connection, it is moved into quarantine. No more
+request will be routed to this container for some times.
```

Comment:
for some times is not clear enough.

We'll try to reach the app with an **exponential backoff**, if it answers back it gets out